lib/pbench/server/api/__init__.py

```diff
@@ -75,6 +99,160 @@ def create_app():
     return app
 
 
+def _build_cors_preflight_response():
+    response = make_response()
+    response = _corsify_actual_response(response)
+    response.headers.add("Access-Control-Allow-Headers", "*")
+    response.headers.add("Access-Control-Allow-Methods", "*")
+    return response
+
+
+def _corsify_actual_response(response):
+    response.headers.add("Access-Control-Allow-Origin", "*")
+    return response
+
+
+class CorsOptionsRequest:
+    """
+    From the client side, an OPTIONS request is initiated before every POST to request CORS.
+    On the server side, the server must specify the response headers to grant CORS accordingly.
+    """
+
+    def __init__(self, options_exception_msg):
+        self.options_exception_msg = options_exception_msg
+
+    def options(self):
+        try:
+            response = _build_cors_preflight_response()
+        except Exception as e:
+            app.logger.exception(
+                self.options_exception_msg + " Raised: %s", e.__class__
```

Comment:
No need reach into the exception class to get the name, just use `self.options_exception_msg` here.